### PR TITLE
Feature: Setup postman

### LINKED ISSUE #77: Setup postman
As a developer I want to develop the REST API with postman. Therefore we have to connect it with github and add a api

## (Wiki) References
- https://github.com/BP2022-AP1/bp2022-ap1/wiki/REST-API

## Acceptance Criteria
- [x] Install postman integration to github
- [x] Setup empty API in postman
- [x] Create shared postman account
- [x] Update wiki

### PR DESCRIPTION
**Pull request by antonykamp**
Fixes #77 

After adding the postman integration to github it adds an empty API and an empty OpenAPI3 definition file.

## PR checklist

- [x] Acceptance criteria fulfilled
- [x] Docs (code, wiki & diagrams) updated
- [x] Breaking changes anounced
- [x] Dev-branch has been merged into local branch to resolve conflicts
- [x] Tests and linter have passed AFTER local merge
- [x] Another dev reviewed and approved

### CODE REVIEW

**Review by Lietze — COMMENTED** [review]
This looks good. Was adding the Postman tests to the CI also part of this Issue? If so, where can I find those changes?

**Comment by antonykamp** [comment]
It's a great question @Lietze 👍 I didn't consider adding it to the pipeline 😄 It's a great idea!
Let's use https://learning.postman.com/docs/collections/using-newman-cli 
I would add a new issue to the next sprint for setting up newman :) see #100

**Review by Lietze — APPROVED** [review]
With the follow-up issue I think this is great.

File: .postman/api [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY
+apis[] = {"apiId":"b23db1c0-fe6b-4c8f-887a-2c9ca66ebf20"}
```
Comment (SaturnHafen) [inline .postman/api:2]: I hope this is not some kind of private identifier :thinking:

**Review by SaturnHafen — APPROVED** [review]
Looks good so far! Really not much to review :slightly_smiling_face:

File: .postman/api (hunk 1 above)
Comment (antonykamp) [reply .postman/api:2]: I like your caution 😄 The commits of this PR are generated out of postman. I hope they don't commit their own private identifier. I think it's just a simple identifier.

**Pull request merged**